pkg/controller/networkpolicy/networkpolicy_crd_utils.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020 Antrea Authors
```

Comment:
maybe just utils.go or crd_utils.go? since it is already in networkpolicy pkg

Reply:
Done